rgbdscheck.asm

```diff
@@ -1,16 +1,6 @@
-MAJOR EQU 0
-MINOR EQU 6
-PATCH EQU 0
-
-WRONG_RGBDS EQUS "fail \"pokered requires rgbds v0.6.0 or newer.\""
-
 IF !DEF(__RGBDS_MAJOR__) || !DEF(__RGBDS_MINOR__) || !DEF(__RGBDS_PATCH__)
-	WRONG_RGBDS
-ELSE
-IF (__RGBDS_MAJOR__ < MAJOR) || \
-	(__RGBDS_MAJOR__ == MAJOR && __RGBDS_MINOR__ < MINOR) || \
-	(__RGBDS_MAJOR__ == MAJOR && __RGBDS_MINOR__ == MINOR && __RGBDS_PATCH__ < PATCH) || \
-	(__RGBDS_MAJOR__ == MAJOR && __RGBDS_MINOR__ == MINOR && __RGBDS_PATCH__ == PATCH && DEF(__RGBDS_RC__))
-	WRONG_RGBDS
-ENDC
+	fail "pokered requires rgbds v0.7.0 or newer."
 ENDC
+IF __RGBDS_MAJOR__ == 0 && __RGBDS_MINOR__ <  7
+	fail "pokered requires rgbds v0.7.0 or newer."
+ENDC
```

Comment:
I know i'm nit picky.. but can we add a new line at EOF?

Reply:
Good catch, thank you!